docs/development/project-setup/environment-variables.md

```diff
@@ -41,6 +41,17 @@ SMTP_USERNAME=
 SMTP_PASSWORD=
 ```
 
+If you would like to use Sendgrid to send your emails, use the following environment variables:
+```bash
+SMTP_ADDRESS=smtp.sendgrid.net
+SMTP_AUTHENTICATION=plain
+SMTP_DOMAIN=<your verified domain>
+SMTP_ENABLE_STARTTLS_AUTO=true
+SMTP_PASSWORD=<your Sendgrid API key>
+SMTP_PORT=587
+SMTP_USERNAME=apikey
```

Comment:
Not sure if these need to be encased in quotes or not since I did it through the Heroku config panel which coerces them to strings anywas.

Reply:
Quotes are not required.

Reply:
really need to add these ``` quates?not able to send email when agent added please help

Reply:
is really need to add these ``` quotes ? becoz not able to send email when agent got added.i have added verified domain and key still not able to send mail please help